lib/emailer.js

```diff
@@ -4,16 +4,18 @@ const path = require('path');
 const nodemailer = require('nodemailer');
 const stubTransport = require('nodemailer-stub-transport');
 const smtpTransport = require('nodemailer-smtp-transport');
+let transport;
```

Comment:
Shouldn't we scope this variable to the instance? If for some reason an application requires 2 instances of the emailer (perhaps on different journeys) this var will be cached from the last time the module was required